### ISSUE TITLE
minor parameter issue

### ISSUE THREAD

**Issue by edwardsmarkf** [agershun/alasql#818]
hello - 

using objects as  parameters does not appear to work for me.  i attempted to use the syntax found here:

https://github.com/agershun/alasql/wiki/Getting%20started

using this syntax:

     alasql.exec('INSERT INTO one (:a, :b);', {a:3,b:4});

http://jsfiddle.net/mtt9r7rx/7/

**Comment by mathiasrw** [comment]
Crazy - we lost some syntax convenience somehow. 

Its not working because its missing `VALUES` - but the example worked in earlier versions. 

This feature is now part part of the regression tests and is implemented. 

Until next release gets out you can use https://github.com/agershun/alasql/blob/develop/dist/alasql.fs.js

**Comment by edwardsmarkf** [comment]
hello - 

should i be able to use objects in an UPDATE as well?
```
alasql.exec('UPDATE one SET :a WHERE :b',  {a: 987654321, b: 4} ) ;
```

http://jsfiddle.net/mtt9r7rx/16/

**Comment by mathiasrw** [comment]
The changes are now released as part of version 0.3.7
